packages/markdown/src/MarkdownEditor.tsx

```diff
@@ -47,11 +47,13 @@ export interface MarkdownEditorProps {
 export function MarkdownEditor(
   props: MarkdownEditorProps & {
     disabled: boolean;
-    initialValue: string | null | undefined;
+    value: string | null | undefined;
     saveValueToSDK: Function;
+    externalReset?: number;
   }
 ) {
-  const [currentValue, setCurrentValue] = React.useState<string>(props.initialValue ?? '');
+  const initialExternalReset = React.useRef(props.externalReset);
```

Comment:
Nit: as you keep this up-to-date,
```suggestion
  const prevExternalReset = React.useRef(props.externalReset);
```